### ISSUE TITLE
Build failure with ghc 8.4

### ISSUE THREAD

**Issue by DanBurton** [juhp/cabal-rpm#55]
Cabal 2.2, in particular

```
[ 7 of 16] Compiling PackageUtils     ( src/PackageUtils.hs, dist/build/cabal-rpm/cabal-rpm-tmp/PackageUtils.o )

src/PackageUtils.hs:68:1: error:
    Could not find module ‘Distribution.PackageDescription.Parse’
    Perhaps you meant
      Distribution.PackageDescription.Parsec (from Cabal-2.2.0.0)
      Distribution.PackageDescription.Check (from Cabal-2.2.0.0)
      Distribution.PackageDescription.Quirks (from Cabal-2.2.0.0)
    Use -v to see a list of the files searched for.
   |
68 | import Distribution.PackageDescription.Parse (readPackageDescription)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by juhp** [comment]
Thanks!  This should be fixed with 0.12.2: I will add it back to Stackage Nightly.

**Comment by juhp** [comment]
I added it back again ;-)
